build_configs/macos/build.spec

```diff
@@ -4,12 +4,15 @@ block_cipher = None
 
 
 a = Analysis(['../../eth2deposit/deposit.py'],
-             binaries=[('/System/Library/Frameworks/Tk.framework/Tk', 'tk'), ('/System/Library/Frameworks/Tcl.framework/Tcl', 'tcl')],
-             datas=[('../../eth2deposit/key_handling/key_derivation/word_lists/*.txt', './eth2deposit/key_handling/key_derivation/word_lists/')],
+             binaries=None,
+             datas=[
+                 ('../../eth2deposit/key_handling/key_derivation/word_lists/*.txt', './eth2deposit/key_handling/key_derivation/word_lists/'),
+                 ('../../eth2deposit/intl', './eth2deposit/intl',)
+             ],
              hiddenimports=[],
              hookspath=[],
              runtime_hooks=[],
-             excludes=[],
+             excludes=['FixTk', 'tcl', 'tk', '_tkinter', 'tkinter', 'Tkinter'],
```

Comment:
MacOS needs this line so that the `Tk` & `Tcl` binaries don't need to be imported as per the change to [L7](https://github.com/ethereum/eth2.0-deposit-cli/pull/182/files/a7facf484c8874d6f1d3c2e37727331cfac9725e#diff-8ebc05df81d902c982839d6583795740de8efcd17ab87692907861346e6b6ab3L7) above.

I don't actually know whether PyInstaller bundles these two binaries into the Windows and Linux builds. I will investigate as removing them could shrink the final binary size a bit.

Reply:
`Tk` and `Tcl` now both explicitly removed from Windows and Linux binaries. No noticeable difference to the size of the exported binaries, but keeping that line in there for consistency's sake.